packages/babel-plugin-transform-es2015-block-scoping/src/index.js

```diff
@@ -459,8 +459,8 @@ class BlockScoping {
     this.hoistVarDeclarations();
 
     // turn outsideLetReferences into an array
-    const params = values(outsideRefs);
     const args = values(outsideRefs);
+    const params = args.map(id => t.clone(id));
```

Comment:
Are there other places we would need to make this change (where we don't clone)?